src/openapi.yaml

```diff
@@ -199,6 +199,8 @@ paths:
     $ref: './resources/EnvironmentStop.yaml'
   /environment/{environmentId}/restart:
     $ref: './resources/EnvironmentRestart.yaml'
+  /environment/{environmentId}/cancelDeployment:
```

Comment:
@erebe should not be able to delete a specific commit id instead of a random environment deployment? BTW is it really possible to delete an environment deployment? 🤔

Reply:
not sure to understand, this endpoint is to cancel deployment at env lvl, you can't specify a commit ID, as an env is a bundle of multiple services.

regarding specigying an ID, we can, but other similar endpoint don't do it also. So it implicitly says, please cancel the top one/the one running ATM

Reply:
ok looks good to me.